### PR TITLE
0.7.0 Added missing netcore dependencies

### PR DESCRIPTION
**Pull request by ncave**
Fixed the netcore build so `build FableCompilerNetcore` is passing again.

### CODE REVIEW

File: src/tests/fableconfig.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
             "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
             "module": "commonjs",
-            "coreLib": "../../build/fable-core/umd",
+            "coreLib": "../../build/fable-core",
```
Comment (alfonsogarciacaro) [inline src/tests/fableconfig.json]: Is this necessary to pass the tests? The default fable-core distribution now is in ES2015 module format, which is not understandable by node. That's why the tests use the UMD (universal module definition) distribution.

Comment (ncave) [inline src/tests/fableconfig.json]: Yes, it's about location, when fable-core is compiled it goes to /build/fable-core/, not /build/fable-core/umd.

Comment (alfonsogarciacaro) [inline src/tests/fableconfig.json]: Ah, ok! I didn't remember that for the `FableCompilerNetcore` target, fable-core TS files were compiled directly in commonjs format. Thanks for the explanation! :+1:

Comment (alfonsogarciacaro) [inline src/tests/fableconfig.json]: Great! I added you as collaborator to the `fable-compiler-netcore` pkg so now you've publish rights :+1: Don't worry about the output of fable-core, I think both versions of fable-compiler can share the same fable-core dependency, so building fable-core in `FableCompilerNetcore` FAKE target is just to pass the tests.

There's already a `PublishCompilerNetcore` target that should work (once you add your npm credentials to your machine, check the npm site for that), it just runs `npm publish` from the build/fable dir (adding `--tag next` if needed). For fable-compiler I do the builds in AppVeyor and there I remove the *.pdb and *.xml files to make the package lighter (check `MakeArtifactLighter` target), you may want to do that as well.

Please tell me if you've any question, and thanks a lot for your help!

**Pull request merged**